src/js/view/controls/rightclick.js

```diff
@@ -167,24 +167,24 @@ export default class RightClick {
         clearTimeout(this.longPressTimeout);
     }
 
-    addListeners() {
-        if (!OS.iOS) {
+    addHideMenuHandlers() {
+        this.removeHideMenuHandlers();
```

Comment:
This is setting `oncontextmenu` to null, and it is never reset. So for Android Chrome, the right-click menu only shows up the first time you long press. Any subsequent long-press will open the default context menu.

Reply:
thanks